### ISSUE TITLE
Add DvD

### ISSUE THREAD

**Issue by jparkerholder** [adaptive-intelligent-robotics/QDax#66]
It would be nice to add Diversity via Determinants (DvD) to the repo. It is possible some of the ideas here could combine well with others to create exciting new QD algorithms :)

Original Paper:
https://arxiv.org/abs/2002.00632 

Could use the implementation from: 
https://arxiv.org/abs/2206.08888

**Comment by Aneoshun** [comment]
Agreed! Would you be able to help us with it, e.g., testing our implementation?

**Comment by jparkerholder** [comment]
Hey - yes I should have some time next week :)

**Comment by felixchalumeau** [comment]
Hey @jparkerholder 

We discussed it in our last development meeting and will start working on it in a matter of days and will keep you updated!

We will implement it in the way made in [Fast PBLR](https://arxiv.org/abs/2206.08888) :fast_forward: :fast_forward:

**Comment by jparkerholder** [comment]
Hey @felixchalumeau, this sounds great! The results look as expected in the paper so it is probably implemented far better than I would have done :)

I am looking forward to trying this, there are a few cool directions to go with this approach and your repo will make it possible!
